### ISSUE TITLE
Bootstrapping is clunky at present

### ISSUE THREAD

**Issue by kennygrant** [fragmenta/fragmenta#28]
Bootstrapping (create db etc) requires the current user to have superuser privileges - this is not the default setup for psql, and it may be desirable anyway to adopt a different driver as the default in order to make setup painless without installing a database locally. 

Need to consider how best to improve bootstrapping. Some options: 

* Look at an embedded db like ql as default 
* Require manual setup of the db before creating tables (still issues with table create permission being required)
* Require separate steps for initially bootstrap (manual), vs further migrations (via db driver)

**Comment by sybrenstuvel** [comment]
I agree, I had to add this to the Fragmenta source just to figure out what's going wrong with `fragmenta migrate`:

```
diff --git a/migrate.go b/migrate.go
index 3609918..595109e 100644
--- a/migrate.go
+++ b/migrate.go
@@ -68,6 +68,9 @@ func migrateDB(config map[string]string) {
                        if err != nil || strings.Contains(string(result), "ERROR") {
                                if err == nil {
                                        err = fmt.Errorf("\n%s", string(result))
+                               } else {
+                                       // If 'result' is not going to be part of the error, log it.
+                                       log.Print(string(result))
                                }
 
                                // If at any point we fail, log it and break
```

The error was `psql: FATAL:  database "sybren" does not exist`, which indicates that the initial migration wants to use the wrong database; it apparently doesn't look at `secrets/fragmenta.json` at all.
